test/run_tests.py

```diff
@@ -17,9 +17,19 @@ def setUp(self):
 
         for _file in self.jsonFiles:
             fpath = os.path.join(self.path + '/test', _file)
+            input_json = None
+            expected_output = None
+            with open('%s.json' % fpath, 'r') as f:
```

Comment:
sidenote here: the `with` statement closes open files when the scope is finished, which avoids warnings in python3.